frontend/src/features/public-form/PublicFormProvider.tsx

```diff
@@ -876,8 +876,7 @@ export const PublicFormProvider = ({
             'features.publicForm.components.saveDraft.toast.restoredOnlyUnchangedFields',
           )
         : t('features.publicForm.components.saveDraft.toast.restoredAllFields')
-      console.log({
-        message: restoreDraftMessage,
+      datadogLogs.logger?.log(restoreDraftMessage, {
```

Comment:
suggestion(minor): wondering if we need the `?` optional chaining operator - since TS gives us assurances that the logger is defined. for example, usage of `datadogLogs.logger` below do not use this. perhaps we should remove this?

Reply:
apart from this LGTM! thanks for taking this on! 🙇

Reply:
Nice catch! Yes, it was a vestigial remainder from using `getLogger()` at first.

```suggestion
      datadogLogs.logger.log(restoreDraftMessage, {
```